robottelo/performance/perf_concurrent_test.py

```diff
@@ -0,0 +1,423 @@
+"""Test utilities for writing performance tests
+
+Define ConcurrentTestCase as parent class of writting Subscribe by AK,
+Subscribe by register and attach, Subscription Deletion.
+
+"""
+import csv
+
+from robottelo.common import conf, ssh
+from robottelo.test import TestCase
+
+from robottelo.performance.perf_thread import DeletePerfThread
+from robottelo.performance.perf_thread import SubscribeAKPerfThread
+from robottelo.performance.perf_thread import SubscribeAttachPerfThread
+from robottelo.performance.perf_stat import generate_stat_for_concurrent_thread
+
+
+class ConcurrentTestCase(TestCase):
```

Comment:
As this is a base TestCase class, it can be moved to the robottelo.test module and live with the other base TestCase classes.